swap_meet/vendor.py

```diff
@@ -1,2 +1,96 @@
+from queue import PriorityQueue
+from swap_meet.item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory  = None):
+        if not inventory:
+            inventory = []
+        self.inventory = inventory
+        
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self,item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
```

Comment:
You could do away with this `else` block and pull that logic into a guard clause at the top of the method. Then you wouldn't need to check whether the `item` is in `self.inventory`. I also want to point out that there's a loop on line 19 that's not doing anything, although it does represent the logic you'd want to use in the guard clause.